mm2src/coins/eth/eth_tests.rs

```diff
@@ -1353,3 +1353,172 @@ fn test_sign_verify_message() {
         .unwrap();
     assert!(is_valid);
 }
+
+#[test]
+fn test_eth_extract_secret() {
```

Comment:
Is this test related to validation of the other pubkey?

Reply:
as there was not any test for `extract_secret` in etc module I decided to add one..should I remove?

Reply:
No, let's keep it, thanks!